recipes/backward-cpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(defs={'CMAKE_VERBOSE_MAKEFILE': 'ON'})
```

Comment:
```suggestion
        cmake.verbose = True
```
https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#verbose

Reply:
I'll remove any verbosity once the recipy has passed all ci + review cycle.

Reply:
okay :+1: